component-catalog/src/Examples/TextInput.elm

```diff
@@ -466,10 +471,21 @@ update msg state =
 
         SetShowPassword showPassword ->
             ( { state | showPassword = showPassword }
-            , Cmd.none
+            , Cmd.batch
+                [ Task.attempt Blurred (Dom.blur "text-input__newPassword-example__visibility-toggle")
```

Comment:
I wonder if we could expose a helper from TextInput so that we don't need to worry about this string falling out of sync?